applications/Web3Box.md

```diff
@@ -0,0 +1,112 @@
+# Web3Box
+- **Team Name:** Web3Box Labs
+- **Payment Address:** 0x2755213c8435F15D9929101357686a9C1Bec280A (USDC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+
+## Project Overview
+
+### Overview
+
+**The dashborad client for all assets management and display of the Polkadot Ecology.**
+
+Follow-up: The client with parachains risk assessment, including risk score, risk report, and risk alert for Polkadot Ecology.
+
+
+
+### Project Details
+
+#### Key Modules ####
+1. Dashboard/Multi-chain Wallet
+- Relay chains & Parachains Assets Management
+- Relay chains & Parachains Assets Portfolio
+
+Web3Box Dashboad Preview
+**![](https://lh3.googleusercontent.com/KZwZrmrRtQHMPqqcGkssfYtvC8SDmvGwB0WjbQ6r12Doqrx5yGyLcraGbVbqSFpYQjR-NcHUUK6tLYfmuq3huMCAsRHT0T_qiQ_AIV-KDvgroW1boEHsbeipCwXi245WDXsP3-LnI5BzpsVcXM4O6ux2RUrujJrY8Ah6qemsCDsxXz6FVLEtICp3JF79qw)**
+
+2. Risk assessment (follow-up grant)
+- Parachains Projects Scoring
+- Parechains Projects Risk Report
+- Parachains Abnormal warning
```

Comment:
I believe @keeganquigley already asked about this, but I'm not quite clear how the risk assessment module would work. You're saying basically that it will rely on project scoring, risk report and abnormal warnings, but what do these terms actually mean and how would you source that data?